Makefile

```diff
@@ -951,7 +951,111 @@ ifeq "$(OPENMP)" "true"
 endif
 
 
-pio_test:
+openacc_test : 
```

Comment:
It looks like there's a stray space at the end of this line. Also, for consistency with other make targets, should we remove the space before the `:`.